build/dsc/manager.rb

```diff
@@ -210,9 +211,24 @@ def document_types(markdown_file_path, dsc_types)
           end
           file.write("\n")
         end
+        
+        gitref = get_powershell_dsc_git_reference        
+        file.write("\nThe vendored Powershell DSC Resource can be rebuilt using `DSC_REF=#{gitref}`") if gitref
```

Comment:
I would prefer to see this at beginning of type file. if we put it at the end, then each time we get a new DSC Resource we move this line.